cmake/kokkoskernels_tpls.cmake

```diff
@@ -388,7 +388,7 @@ KOKKOSKERNELS_ADD_TPL_OPTION(MAGMA    OFF "Whether to enable MAGMA")
 
 # Set F77_BLAS_MANGLE macro based on Fortran-C interface (unless already set
 # by Trilinos or user)
-IF (F77_BLAS_MANGLE STREQUAL "")
+IF ("${F77_BLAS_MANGLE}" STREQUAL "")
```

Comment:
Alright, wanted to confirm where the typo was last time - the closing brace got placed after the STREQUAL

`IF ("${F77_BLAS_MANGLE STREQUAL}" "")`

Much better :)